src/Makefile.test.include

```diff
@@ -29,8 +29,6 @@ JSON_TEST_FILES = \
   test/data/tx_valid.json \
   test/data/sighash.json
 
-RAW_TEST_FILES = test/data/alertTests.raw
```

Comment:
Here you are deleting the only `RAW_TEST_FILE` definition - please remove it in the rest of the file or leave this variable empty here to not cause confusion (and keep it for the future).